### DISCUSSION TITLE
Dynamic Landscape - Error code 36

### DISCUSSION THREAD

**Discussion by S1427** [RangeShifter/RangeshiftR-tutorials#52] [Q&A]
Hello, I am having a problem with the dynamic landscape settings.  

In the Outputs folder : 

```
Event, Number, Reps, Years, Time
RNG SEED,,,,1723018577
Landscape,0,ERROR,CODE,36

Batch,,,,7.
``` 

What does error code 36 mean? Can it explain aborting of reading landscape #0?

Thank you in advance for your help !

**Comment by JetteReeg** [comment]
Hi, the error code 36 means `Invalid 'no data' cell read from habitat file in readLandChange()`. I would first compare all landscape input files. You can find the list of error codes [here](https://github.com/RangeShifter/RangeShifter_batch/blob/main/doc/Manual/Batchmode/Batch_error_codes.xlsx).
